src/otk/command.py

```diff
@@ -116,13 +118,22 @@ def compile(
     tree = resolve(spec, tree[f"{PREFIX_TARGET}{kind}.{name}"])
 
     # and then output by writing to the output
-    dst.write_text(
-        target_registry.get(kind, CommonTarget)().as_string(spec, tree)
-    )
+    if not dry_run:
+        dst.write_text(
+            target_registry.get(kind, CommonTarget)().as_string(spec, tree)
+        )
 
     return 0
 
 
+def compile(parser: argparse.ArgumentParser, arguments: argparse.Namespace) -> int:
+    return _process(parser, arguments, False)
```

Comment:
Maybe change this to `return _process(parser, arguments, dry_runFalse)` (and similar below)? This way it becomes easier to immediately see what the "true/false" mean in here?